keyboards/order/keymaps/default/keymap.c

```diff
@@ -0,0 +1,72 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
+
+  /* Qwerty
+   * .--------------------------------------------------------------.
+   * |ESC |1  |2  |3  |4  |5  |6  |7  |8  |9  |0  |-  |=  |bkspc|F13|
+   * |--------------------------------------------------------------+
+   * |Tab |Q  |W  |E  |R  |T  |Y  |U  |I  |O  |P  |[  |]  |\    |F14|
+   * |--------------------------------------------------------------+
+   * |Cap |A  |S  |D  |F  |G  |H  |J  |K  |L  |;  |'  |Ent|PgUp|F15 |
+   * |--------------------------------------------------------------+
+   * |Shi |Z  |X  |C  |V  |B  |N  |M  |,  |.  |/  |Shi|Up |Pgdn|F16 |
+   * |--------------------------------------------------------------+
+   * |Ctr|Win|Alt|cop|pas|Spc|   |   |Alt |FN |Ctr|L  |D  |R   |F17 |
+   * '--------------------------------------------------------------'
+   */
+
+  /* FnLayer
+   * .--------------------------------------------------------------.
+   * | ` ~ |F1 |F2 |F3 |F4 |F5 |F6 |F7 |F8 |F9 |F10|F11|F12|   |   |
+   * |--------------------------------------------------------------+
+   * | Tab |   |   |   |   |   |   |   |   |   |   |   |   | \ |   |
+   * |--------------------------------------------------------------+
+   * | Caps|   |   |   |   |   |   |   |   |   |   |   |Ent|   |   |
+   * |--------------------------------------------------------------+
+   * | Shi|    |   |   |   |   |   |   |   |   |   |   |Shi|VUP|   |
+   * |--------------------------------------------------------------+
+   * | Ctrl|Win|Alt|cop|pas|   |   |   |   |Alt|FN |Ctrl|  |VDN|   |
+   * '--------------------------------------------------------------'
+   */
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(
+      KC_GESC ,KC_1     ,KC_2     ,KC_3     ,KC_4     ,KC_5     ,KC_6     ,KC_7     ,KC_8     ,KC_9     ,KC_0     ,KC_MINS  ,KC_EQL   ,KC_BSPC  ,KC_F13 ,
+      KC_TAB  ,KC_Q     ,KC_W     ,KC_E     ,KC_R     ,KC_T     ,KC_Y     ,KC_U     ,KC_I     ,KC_O     ,KC_P     ,KC_LBRC  ,KC_RBRC  ,KC_BSLS  ,KC_F14 ,
+      KC_CAPS ,KC_A     ,KC_S     ,KC_D     ,KC_F     ,KC_G     ,KC_H     ,KC_J     ,KC_K     ,KC_L     ,KC_SCLN  ,KC_QUOT  ,KC_ENT   ,KC_PGUP  ,KC_F15 ,
+      KC_LSFT ,KC_Z     ,KC_X     ,KC_C     ,KC_V     ,KC_B     ,KC_N     ,KC_M     ,KC_COMM  ,KC_DOT   ,KC_SLSH  ,KC_RSFT  ,KC_UP    ,KC_PGDN  ,KC_F16 ,
+      KC_LCTL ,KC_LGUI  ,KC_LALT  ,KC_COPY  ,KC_PASTE ,KC_SPC   ,         ,         ,KC_RALT  ,MO(_FL)  ,KC_RCTL  ,KC_LEFT  ,KC_DOWN  ,KC_RIGHT ,KC_F17),
```

Comment:
You need to pass an argument between each pair of commas.

```suggestion
      KC_LCTL ,KC_LGUI  ,KC_LALT  ,KC_COPY  ,KC_PASTE ,KC_SPC   ,KC_NO    ,KC_NO    ,KC_RALT  ,MO(_FL)  ,KC_RCTL  ,KC_LEFT  ,KC_DOWN  ,KC_RIGHT ,KC_F17),
```